### PR TITLE
util: fix cpt upgrader for rvv changes in PR #83

### LINKED ISSUE #106: "util/cpt_upgrader.py" not updating RISC-V checkpoints from v23.0.0.1 to changes in #83
To reproduce:

```sh
# Obtain the gem5 repo
git clone https://github.com/gem5/gem5.git
cd gem5

# Obtain the #83 PR
git fetch origin pull/83/head:rvv
git switch rvv

wget http://dist.gem5.org/dist/develop/checkpoints/riscv-hello-example-checkpoint-20230222.tar
mkdir checkpoint
tar xf riscv-hello-example-checkpoint-20230222.tar -C checkpoint/
./util/cpt_upgrader.py -v checkpoint
```

The following output is returned:

```sh
Processing file checkpoint/m5.cpt.... 

has tags arm-miscreg-teehbr arm-ccregs arm-gem5-gic-ext dvfs-perflevel x86-add-tlb ide-dma-abort remove-arm-cpsr-mode-miscreg etherswitch armv8 register-files ruby-block-size-bytes mempool-to-seworkload process-fdmap-rename cpu-pid globals-to-root smt-interrupts memory-per-range mempool-sections isa-is-simobject arm-sysreg-mapping-ns arm-contextidr-el2 multiple-event-queues arm-hdlcd-upgrade arm-sve arm-gicv2-banked-regs 

...nothing to do 
```

I have only proven this fault when working with #83 for updating a checkpoint from v23.0.0.1 to that necessary for the PR. The limitation of "util/cpt_upgrader.py" may exist elsewhere.

### PR DESCRIPTION
**Pull request by aarmejach**
Solves issue #106 by updating the cpts with the necessary vector registers.

Change-Id: Ifeda90e96097f0b0a65338c6b22a8258c932c585

### CODE REVIEW

**Review by powerjg — COMMENTED** [review]
Thanks! Should this be on top of PR #83?

**Comment by aarmejach** [comment]
hey Jason, I think this should be merged right after PR #83 to make things consistent.

BTW I did test this works by taking the checkpoint of the failing test, applying the cpt_upgrader with this change and checking the test passes on the updated checkpoint.

**Review by BobbyRBruce — DISMISSED** [review]
Looks good to me. Thanks! We'll wait for the RVV PR to be merged before this.

**Comment by aarmejach** [comment]
Updated given changes on PR #83

File: util/cpt_upgraders/riscv-vext.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+def upgrader(cpt):
```
Comment (powerjg) [inline util/cpt_upgraders/riscv-vext.py:28]: This needs a license. Sorry for not noticing until now.

**Review by powerjg — CHANGES_REQUESTED** [review]

File: util/cpt_upgraders/riscv-vext.py (hunk 1 above)
Comment (aarmejach) [reply util/cpt_upgraders/riscv-vext.py:28]: done!

**Review by powerjg — APPROVED** [review]

**Review by BobbyRBruce — APPROVED** [review]

**Pull request merged**